BayeuxClient.agent.lib.nut

```diff
@@ -858,13 +853,15 @@ class Bayeux.Client {
             return;
         }
 
-        // TODO: Handle the cookies properly
-        // Waiting for EI to fix the issue with Set-Cookie headers
-        foreach (k, v in response.headers) {
-            if (k.tolower() == "set-cookie") {
+        foreach (header in response.rawheaders) {
+            if (header.k.tolower() == "set-cookie") {
                 try {
-                    local cookieName = split(v, "=")[0];
-                    _cookies[cookieName] <- v;
+                    // Cookie's value should be between the first "=" and the first ";"
+                    local startIdx = header.v.find("=");
```

Comment:
does the standard allow for spaces around `=`?

Reply:
Spaces are forbidden but now we handle this, just in case